config/v1/types_scheduling.go

```diff
@@ -28,6 +28,15 @@ type SchedulerSpec struct {
 	// The namespace for this configmap is openshift-config.
 	// +optional
 	Policy ConfigMapNameReference `json:"policy"`
```

Comment:
Do we want to start the deprecation on this one or we'll wait with it?

Reply:
All the flags that use this have been marked deprecated upstream. The API itself isn't technically marked as "deprecated" yet upstream but I think it would be good to signal that this is going to be deprecated if we can.

Reply:
Before we proceed with this approach, we need to have a broader discussion what we want to expose. Looking at https://github.com/kubernetes/kubernetes/blob/4c853bb28f57f87fbb2c1c6f2845c701d90c2350/staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go#L44-L99, which is the scheduler component config, I'm not sure I want to expose this entirely to users. IIRC the approach we've been taking in KCM and KAS was NOT to expose the underlying configs directly, but rather provide higher-level knobs which then translate to actual flags in the component. @deads2k can prove me wrong, but let's talk this through during next planning before continuing with this.